languages/csharp/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -19,13 +19,13 @@ In this exercise you must conditionally execute logic. The most common way to do
 ```csharp
 int x = 6;
 
-if (x <= 5)
+if (x == 5)
 {
-    // Execute logic if x <= 5
+    // Execute logic if x equals 5
```

Comment:
Would it be too pedantic to say this need to be more specific ?
e.g.  `//Execute logic if the value of x equals 5` (same for other similar comments)

Reply:
Ehm, maybe :) I try to keep these comments as short as possible, as it lessens the chance of people skipping them. Do you feel that it is too short?

Reply:
Not too short. I tend to err on the side of being specific. But if you think that is too verbose then that would not be good either. Maybe ignore my initial comment here 😄

Reply:
:)